x-pack/legacy/plugins/lens/public/editor_frame_plugin/merge_tables.ts

```diff
@@ -38,16 +42,35 @@ export const mergeTables: ExpressionFunction<
     },
   },
   context: {
-    types: ['null'],
+    types: ['kibana_context', 'null'],
   },
-  fn(_ctx, { layerIds, tables }: MergeTables) {
+  fn(ctx, { layerIds, tables }: MergeTables) {
     const resultTables: Record<string, KibanaDatatable> = {};
     tables.forEach((table, index) => {
       resultTables[layerIds[index]] = table;
     });
     return {
       type: 'lens_multitable',
       tables: resultTables,
+      dateRange: getDateRange(ctx),
```

Comment:
This does look like it'll work, but is there a simpler way? Is it possible that each of the nested expressions already has access to the context?

```
kibana_context
| lens_multitable tables={expression1} tables={expression2}
```

Reply:
I think we'll want to address the Kibana context thing more thoroughly in a future release, but for now, this is the easiest way I could find to pass this information through.